modules/dop/services/autotest/filetree_copy.go

```diff
@@ -110,7 +110,9 @@ func (svc *Service) CopyFileTreeNode(req apistructs.UnifiedFileTreeNodeCopyReque
 		}
 	}()
 
-	return convertToUnifiedFileTreeNode(dupNode, nil), nil
+	node = convertToUnifiedFileTreeNode(dupNode, nil)
+	go svc.reportPipelineDefinition(*node)
```

Comment:
how about sync failed?
I think synchronous operation is better at begining.

Reply:
It can only be said that it is currently a beta implementation. The consistency problem of synchronization needs to be considered how to solve it, and then how to ensure consistency when using it later.